content/blog/2016/2016-10-11-stage-lock-milestone.adoc

```diff
@@ -122,4 +122,44 @@ Similarly a pair of `milestone` steps used with a `lock` will discard all old bu
 
 === Putting it all together
 
+[source, groovy]
+----
+stage("Build"){
+  // The first milestone step starts tracking concurrent build order
+  milestone
+  node{
+    echo "Building"
+  }
+}
+
+// This locked resource contains both Test stages as a single concurrency Unit.
+// Only 1 concurrent build is allowed to utilize the test resources at a time.
+// Newer builds are pulled off the queue first. The milestone at the end will
+// abort any antecedent build waiting in the queue ensuring that only builds
+// started after the current build remain in the lock queue.
```

Comment:
antecedent?  I understand it, but "previously started".  
Also, "in the queue" - build queue or queue for the resource?